src/main/java/spoon/support/compiler/jdt/JDTImportBuilder.java

```diff
@@ -108,6 +109,13 @@ void build() {
 		spoonUnit.setImports(this.imports);
 	}
 
+	private CtImport createImportWithPosition(CtReference ref, ImportReference importRef) {
+		CtImport imprt = factory.Type().createImport(ref);
+		imprt.setPosition(factory.Core().createCompoundSourcePosition(spoonUnit, importRef.sourceStart(), importRef.sourceEnd(), importRef.declarationSourceStart, importRef.declarationEnd, spoonUnit.getLineSeparatorPositions()));
+		//TODO initialize source position of ref
```

Comment:
I think the position of the reference should be `importRef.declarationSourceStart, importRef.declarationEnd`, no?

Reply:
probably yes, but I do not need it now, so the position of whole import was enough for this PR and sniper mode. I suggest to set position of reference in a later PR.

Reply:
Ok for me.